spec/lib/rbac/filterer_spec.rb

```diff
@@ -2884,6 +2934,54 @@ def id_for_model_in_region(model, other_region)
     end
   end
 
+  # private method
+  describe ".select_from_order_columns" do
+    subject { described_class.new }
+    it "removes empty" do
+      expect(subject.select_from_order_columns([])).to eq([])
+      expect(subject.select_from_order_columns([nil])).to eq([])
+    end
+
+    it "removes string columns" do
+      expect(subject.select_from_order_columns(%w[name id vms.name "vms"."name"])).to eq([])
+    end
+
+    it "removes ascending string columns" do
+      expect(subject.select_from_order_columns(ascenders(%w[name id vms.name "vms"."name"]))).to eq([])
+    end
+
+    # services.name desc (spec)
+    it "removes strings with sorting" do
+      expect(subject.select_from_order_columns(["name asc", "\"vms\".\"id\" desc", "id asc"])).to eq([])
```

Comment:
Would be nice if you were consistent with the array types...

... especially considering you to the trouble of typing this up...  https://github.com/ManageIQ/manageiq/pull/19025#issuecomment-518379296

Reply:
I changed the other one to an array - Rubocop keeps complaining that I didn't use the correct form of an array depending upon having spaces or quotes or what have you

Reply:
in the future, you can just say "rubocop, I disagree"... it's more about detecting things you didn't realize... if you are  purposely going against a cop, just say so and ignore rubocop

Reply:
@kbrock You can also inline ignore if you want, and state the reasoning as part of the comment.  I do that a lot:

https://github.com/ManageIQ/manageiq/blob/efd80502/app/models/manageiq/providers/embedded_ansible/automation_manager/machine_credential.rb#L14-L50

Reply:
I changed the code so the bad code is now a good cop.